.env.development

```diff
@@ -8,5 +8,3 @@ DFE_SIGN_IN_API_CLIENT_ID=teacherpayments
 DFE_SIGN_IN_API_ENDPOINT=https://pp-api.signin.education.gov.uk
 
 ADMIN_ALLOWED_IPS=::1,127.0.0.1
-
-NOTIFY_TEMPLATE_ID=d72e2ff9-b228-4f16-9099-fd9d411c0334
```

Comment:
Do we also need to remove `NOTIFY_TEMPLATE_ID` from the infrastructure `template.json` files?

Reply:
Good point, thanks. Fixed.